### DISCUSSION TITLE
Allow zooming / scaling of live timing windows

### DISCUSSION THREAD

**Discussion by BeyondVertical** [f1multiviewer/issue-tracker#194] [Feature requests]
I would like to have the possibility to scale the whole page (smaller or bigger).
Usage example: when I sit at my computer I could scale it down and see more positions. When I watch it on the TV I probably would scale it up a little to see it clearer from the couch.

**Comment by rriegner** [comment]
I think the whole window also needs size/fit options like the video feeds. Right now its size doesn't fit well within any layout, and resizing the window causes lots of wasted space

**Comment by jeppeolesen** [comment]
Redirecting #194, #583 and #577 here.

**Reply by interi-mhickman** [reply]
Just bringing over my request from #577:
> I use multi viewer to display stats, in-car cameras, and radio transcripts while watching the race on TV. Requesting the ability to increase only the radio transcript panel / window font size to enable it to be readable from across the room. I currently use Ctrl-+ to zoom the text, but this zooms all content across all Multiviewer windows.

**Comment by Post79** [comment]
You can use the font resize function in Windows settings to increase the font size. 

Unfortunately you cannot add css settings or similar to change the LT setup. Would be great to be able to tweak font size and padding for each element. Especially for use on large screens.
